requirements.txt

```diff
@@ -2,5 +2,5 @@ jmespath
 jsonschema
 python-dateutil
 ruamel.yaml
-sphinx
-sphinx-rtd-theme
+sphinx<8
+sphinx-rtd-theme<3
```

Comment:
Doc build breaks on latest version of sphinx/rtd-theme. It does not tell you why, so restrict the version to a lower one for now. Maybe switch to mkdocs in the future.